frontend/src/app/features/quiz/result/result-character/result-character.component.ts

```diff
@@ -0,0 +1,161 @@
+import { CommonModule } from '@angular/common';
+import { Component, Input } from '@angular/core';
+import { SvgIconComponent } from 'angular-svg-icon';
+import { SharedModule } from '../../../../shared/shared.module';
+import { Character } from '../../../../Enum/character';
+import { CharacterAttributesComponent } from './character-attributes/character-attributes.component';
+
+@Component({
+  selector: 'app-result-character',
+  standalone: true,
+  imports: [
+    SharedModule,
+    CommonModule,
+    SvgIconComponent,
+    CharacterAttributesComponent,
+  ],
+  templateUrl: './result-character.component.html',
+  styleUrl: './result-character.component.css',
+})
+export class ResultCharacterComponent {
+  @Input() public character!: Character;
+
+  public bgLink = '';
+  public characterNameEn = '';
+  public characterNameTh = '';
+  public characterTitle = '';
+  public characterImgLink = '';
+  public characterDescription = '';
+  public characterAttributes: string[] = [];
+  public attributesChipColor = '';
+  public shinningMethod = '';
```

Comment:
Saggestion: Use as object in stead of single var ?

Reply:
Done